lib/online_shop-stack.ts

```diff
@@ -1,16 +1,280 @@
-import * as cdk from 'aws-cdk-lib';
-import { Construct } from 'constructs';
-// import * as sqs from 'aws-cdk-lib/aws-sqs';
+import * as cdk from '@aws-cdk/core';
+import * as appsync from '@aws-cdk/aws-appsync';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as lambdaNodejs from '@aws-cdk/aws-lambda-nodejs';
+import * as cognito from '@aws-cdk/aws-cognito';
+import {CfnUserPoolUserToGroupAttachmentProps} from "@aws-cdk/aws-cognito";
+import * as dynamodb from '@aws-cdk/aws-dynamodb';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn_tasks from '@aws-cdk/aws-stepfunctions-tasks';
+import * as sqs from '@aws-cdk/aws-sqs';
 
 export class OnlineShopStack extends cdk.Stack {
-  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
     super(scope, id, props);
 
-    // The code that defines your stack goes here
+    cdk.Tags.of(this).add('Owner', 'shahrukh.khan@trilogy.com');
```

Comment:
Probably shouldn't hard-code this and taken from ENV.

Reply:
done